GameData/RealismOverhaul/Waterfall_Configs/_Processor/_early-initialize.cfg

```diff
@@ -0,0 +1,43 @@
+// Disable Waterfall's engine lights if requested.
+@WATERFALL_SETTINGS:FOR[ROWaterfall]:NEEDS[ROWaterfallNoEngineLight]
+{
+    @EnableLights = false
+}
+
+// Remove RealPlume templates. If a RealPlume template is needed in addition to Waterfall, add it
+// in `:AFTER[ROWaterfall]`.
+@PART[*]:HAS[@ROWaterfall]:FOR[ROWaterfall]:NEEDS[Waterfall]
+{
+    !PLUME,* {}
+    !PLUME_TEMPLATE,* {}
+}
```

Comment:
This seems a little hostile, to force another system to run in a late pass to accomodate this system.  Not sure when the ROWaterfall node is expected to be added to a part, but suggest allowing a tag in there to indicate if RealPlume configs should not be removed.

Reply:
It was my intention to force clear existing RealPlumes so that any RealPlume that coexists with Waterfall needs to have been written with Waterfall in mind. That said, in some rare situations (ex. a part with SRM ullage motors and RCS) this results in needless copy-pasting, so I'll add the tag in.

I feel like `:AFTER[ROWaterfall]` is not a particularly late pass to be adding things -- this patch runs in `:FOR[ROWaterfall]` instead of a `zROWaterfall_xx` pass for that specific use in mind, after all.

The documentation states that `ROWaterfall` nodes are added in `:BEFORE[ROWaterfall]`.

FWIW, RealPlume doesn't have any kind of facility for keeping stock plumes (see https://github.com/KSP-RO/RealPlume/blob/501fb305f619f4f80bcee25efd5e1825f4815a61/GameData/RealPlume/000_Generic_Plumes/000_EFFECTS_Remover.cfg#L63-L66).